File: ESAResturant.NEW.web/ClientApp/src/app/admin/bill-invoice/Models/TableBill.ts
export class TableBill  {

  constructor() {

    this.id = 0;
    this.tableNo = '';
    this.pepoleName = '';
    this.checkWiteInvoies = false;
    this.isDelete = false;
  }
  
  id: number;
  tableNo: string;
  pepoleName: string;
  currentDate: Date;
  date: Date;
  netTotal: number;
  checkWiteInvoies: boolean;
  isDelete: boolean;
}

File: ESC.Resturant.Data/Entities/TableBill.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace ESC.Resturant.Data.Entities
{
    public class TableBill
    {

        [Key]
        public int Id { get; set; }
        public string TableNo { get; set; }
        public string PepoleName { get; set; }
        public Nullable<System.DateTime> CurrentDate { get; set; }
        public Nullable<System.DateTime> Date { get; set; }       
        public Nullable<double> NetTotal { get; set; }    
        public Nullable<bool> CheckWiteInvoies { get; set; }
        public Nullable<bool> IsDelete { get; set; }

    }
}
